test.js

```diff
@@ -350,19 +462,54 @@ exports.test = function (global) {
       }
     }
 
+    function fillGraphVertexes (c, n, g) {
+      print("Filling vertices for ", c.name())
+      for (let i = 0; i < n; ++i) {
+        c.save({
+          _key: "smart" + i + ":test" + i,
+          value1: i,
+          value2: "smart" + i,
+          value3: i,
+          value4: "test" + i,
+          value5: i,
+          value6: "test" + i,
+          value7: i % g,
+          value8: "test" + (i % g)
+        });
+      }
+    }
+
+    function createVertexes(n) {
+      let g = n / 100;
+      fillGraphVertexes(gc.vertex, n, g);
+      fillGraphVertexes(sg.vertex, n, g);
+      if (supportsSatelliteGraphs) {
+        fillGraphVertexes(satg.vertex, n, g);
+      }
+    }
+
+    function createEdges(n) {
+      fillGraphEdges(gc.edges, n, gc.vertex);
+      fillGraphEdges(sg.edges, n, sg.vertex);
+      if (supportsSatelliteGraphs) {
+        fillGraphEdges(satg.edges, n, satg.vertex);
+      }
+    }
+
     if (global.small) {
+      createVertexes(10000);
```

Comment:
```suggestion
      createVertices(10000);
```
😱

Reply:
... I know, but `Vertexes` is AE and arangodb uses AE. :(